examples/stable/postgresql-ha/hook-blueprint/README.md

```diff
@@ -0,0 +1,171 @@
+# PostgreSQL-HA
+
+[PostgreSQL](https://www.postgresql.org/) is an object-relational database management system (ORDBMS) with an emphasis on extensibility and on standards-compliance. 
+
+## Introduction
+
+This Helm chart has been developed based on [Postgresql](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) chart but including some changes to guarantee high availability such as:
+
+- A new deployment, service have been added to deploy [Pgpool-II](https://pgpool.net/mediawiki/index.php/Main_Page) to act as proxy for PostgreSQL backend. It helps to reduce connection overhead, acts as a load balancer for PostgreSQL, and ensures database node failover.
+- Replacing `bitnami/postgresql` with `bitnami/postgresql-repmgr` which includes and configures repmgr. Repmgr ensures standby nodes assume the primary role when the primary node is unhealthy.
+
+Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment and management of Helm Charts in clusters. This chart has been tested to work with NGINX Ingress, cert-manager, fluentd and Prometheus on top of the [BKPR](https://kubeprod.io/).
+
+## Requirements
+When restoring the postgres with high availability in a different namespace, the standby instance pod goes into ``CrashLoopBackOff`` since the connection info for the primary/secondary nodes in the ``repmgr`` database points to source namespace. The blueprint ``postgres-ha-hook.yaml`` can be used to solve this issue which will update the ``repmgr`` database with correct connection information for primary and secondary instances
```

Comment:
```suggestion
When restoring the PostgreSQL with high availability in a different namespace, the standby instance pod goes into ``CrashLoopBackOff`` since the connection info for the primary/secondary nodes in the ``repmgr`` database points to source namespace. The blueprint ``postgres-ha-hook.yaml`` can be used to solve this issue which will update the ``repmgr`` database with correct connection information for primary and secondary instances
```